### PR TITLE
Merge RouteType and RouteTypeExtended

### PR DESCRIPTION
**Pull request by sarahjanenewnham**

### CODE REVIEW

**Review by peetjvv — APPROVED** [review]

File: src/GTFS/Entities/Enumerations/RouteTypeExtended.cs [hunk 1]
```diff
@@ -1,612 +0,0 @@
-﻿// The MIT License (MIT)
```
Comment (timminata) [inline src/GTFS/Entities/Enumerations/RouteTypeExtended.cs:1]: Was this file removed? I'm a super noob at reviews on github, just want to make srue the file was also removed.

File: src/GTFS/Entities/Enumerations/RouteType.cs [hunk 2]
```diff
@@ -28,35 +28,35 @@ namespace GTFS.Entities.Enumerations
     public enum RouteType
```
Comment (timminata) [inline src/GTFS/Entities/Enumerations/RouteType.cs:28]: Looks good, we don't want to rename this for clarity right?

**Review by timminata — APPROVED** [review]

File: src/GTFS/Entities/Enumerations/RouteTypeExtended.cs (hunk 1 above)
Comment (peetjvv) [reply src/GTFS/Entities/Enumerations/RouteTypeExtended.cs:1]: it says it was deleted on my end

File: src/GTFS/Entities/Enumerations/RouteType.cs (hunk 2 above)
Comment (sarahjanenewnham) [reply src/GTFS/Entities/Enumerations/RouteType.cs:28]: I thought changing it to `RouteTypeAll` might be more confusing. Any suggestions?

File: src/GTFS/Entities/Enumerations/RouteTypeExtended.cs (hunk 1 above)
Comment (sarahjanenewnham) [reply src/GTFS/Entities/Enumerations/RouteTypeExtended.cs:1]: Yup. It's deleted.

**Pull request merged**